ActivityHistoryApi/V1/Gateways/ActivityHistoryDbGateway.cs

```diff
@@ -0,0 +1,71 @@
+using Amazon.DynamoDBv2.DataModel;
+using ActivityHistoryApi.V1.Domain;
+using ActivityHistoryApi.V1.Factories;
+using ActivityHistoryApi.V1.Infrastructure;
+using Microsoft.Extensions.Logging;
+using Hackney.Core.Logging;
+using System.Threading.Tasks;
+using Hackney.Core.DynamoDb;
+using ActivityHistoryApi.V1.Boundary.Request;
+using System.Collections.Generic;
+using Amazon.DynamoDBv2.DocumentModel;
+using System.Linq;
+
+namespace ActivityHistoryApi.V1.Gateways
+{
+    public class ActivityHistoryDbGateway : IActivityHistoryGateway
+    {
+        private const int MAX_RESULTS = 10;
+        private const string GETACTIVITIESBYTARGETIDINDEX = "ActivityHistory";
```

Comment:
Not a great index name (see elsewhere).
Also I couldn't find any requirement for ordered results. It makes absolute sense, but I would still validate that.

Reply:
Yeh I thought it would prob be needed in the future so adding it now